config/environments/test.rb

```diff
@@ -1,6 +1,8 @@
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
 
+  config.log_level = ENV.fetch('LOG_LEVEL') { 'info' }.to_sym
```

Comment:
@gigxz, was this intentional?

Reply:
Yeah, sorry I thought I left a comment about this. Without it, there's no way to see what tests are failing on CI because there are too many logs and it refuses to load. I'll move it to `rails_tests.yml`, though, so we can still have debug logs locally by default.